ts/Core/Interaction.ts

```diff
@@ -853,7 +853,9 @@ extend(Chart.prototype, /** @lends Chart.prototype */ {
                             isNumber(dataExtremes.dataMax)
                         ) {
                             panningState.startMin = Math.min(
-                                dataExtremes.dataMin, panningState.startMin
+                                pick(series.options.threshold, Infinity),
```

Comment:
The handling of `startMin` and `startMax` was previously equivalent, but now `startMin` also takes the `threshold` into account. Gut feeling says they should still be equivalent, and the fail case is when all values are negative, so that the `threshold` is the max value. Try to zoom in, then we should expect to be able to pan up to 0: https://jsfiddle.net/highcharts/zupgLn59/